Changelog.md

```diff
@@ -4,6 +4,10 @@ Language Features:
 
 
 Compiler Features:
+ * Commandline Interface: Add ``--ast-compact-json`` option with Yul input (e.g. with ``--strict-assembly``, ``--yul``).
+ * Commandline Interface: Add ``--ir-ast-json`` and ``--ir-optimized-ast-json`` output options which output the Yul-IR and the optimized Yul-IR in AST compact JSON format.
+ * Standard JSON Interface: Support ``irAst`` and ``irOptimizedAst`` options in output selection which output the Yul-IR and the optimized Yul-IR in AST compact JSON format.
+ * Standard JSON Interface: Support the ``ast`` option in output selection with Yul input.
 
```

Comment:
I'm going nitpick these a bit too :)

We should make it clearer what is completely new and what just supported in extra input modes.

```suggestion
 * Commandline Interface: Add ``--ir-ast-json`` and ``--ir-optimized-ast-json`` outputs, providing AST in compact JSON format for IR and optimized IR.
 * Commandline Interface: Support ``--ast-compact-json`` output also in assembler mode.
 * Standard JSON Interface: Add ``ast`` contract-level output for Yul input.
 * Standard JSON Interface: Add ``irAst`` and ``irOptimizedAst`` contract-level outputs for Solidity input, providing AST in compact JSON format for IR and optimized IR.
```